flask/app/routes.py

```diff
@@ -26,6 +30,8 @@ def login():
     if user is None or not user.check_password(body['password']):
         return 'Unauthorized', 401
     login_user(user)
+    user.last_login = datetime.now()
+    db.session.commit()
     return json.dumps({ "username": user.username }), 200
```

Comment:
Do we want to try/except the db commits in case something goes awry or is it acceptable to throw an error 500 when trying to sign in if this fails?

Reply:
Yeah, that's a good point, I'll update

Reply:
I added a new commit, it's just a 500 error to match the other wrapped db.commits in other endpoints but I can update if there's a more specific error code we can show here